src/interfaces/SpokePool.ts

```diff
@@ -111,6 +112,7 @@ export interface RunningBalances {
 
 export interface TokensBridged extends SortableEvent {
   amountToReturn: BigNumber;
+  caller: string;
```

Comment:
Will this impact any downstream events in the relayer?

Reply:
Based on the tests we performed (linking the relayer with this version) it shouldn't. The unit tests of the relayer are passing and both the relayer and the finalizer are working ok in simulation mode.